hera_mc/cm_hookup.py

```diff
@@ -106,21 +105,48 @@ def __get_part_types_found(self, huco, part_types_found):
             part_types_found.append(part_type)
         return part_types_found
 
-    def __get_pols_to_do(self, part, port, check_pol=False):
-        all_pols = ['e', 'n']
-        if port[0].lower() in all_pols:
-            pols = [port[0].lower()]
-        elif port.lower() in ['a', 'b']:
-            pols = [part[-1].lower()]
-        else:
-            pols = all_pols
-        if check_pol is not False:
-            if check_pol in pols:
-                pols = True
+    def __get_pols_to_do(self, part, port_query):
+        """
+        Given the current part and port_query (which is either 'all', 'e', or 'n')
+        this figures out which pols to do.  Basically, given 'all' and part it
+        figures out whether to return ['e'], ['n'], ['e', 'n']
+
+        Parameter:
+        -----------
+        part:  current part dossier
+        port_query:  the ports that were requested.
+        """
+        single_pol_parts_paper = ['RI', 'RO', 'CR']
+        pq = port_query.lower()
```

Comment:
Stylistically, I find it neater to modify the variable in question rather than adding another variable with a similar name: `port_query = port_query.lower()`.